x-pack/plugins/infra/public/components/metrics_explorer/helpers/create_tsvb_link.ts

```diff
@@ -88,9 +88,9 @@ const createFilterFromOptions = (
     filters.push(options.filterQuery);
   }
   if (options.groupBy) {
-    filters.push(`${options.groupBy}: ${series.id}`);
+    filters.push(`${options.groupBy} : "${series.id}"`);
   }
-  return filters.join(' AND ');
+  return { language: 'kuery', query: filters.join(' and ') };
```

Comment:
Oh, so TSVB already supports KQL? TIL

Reply:
Yeah... I was pretty excited to find this too.